helix-term/src/ui/mod.rs

```diff
@@ -286,9 +282,7 @@ pub mod completers {
         let (dir, file_name) = if input.ends_with(std::path::MAIN_SEPARATOR) {
             (path, None)
         } else {
-            let file_name = path
-                .file_name()
-                .map(|file| file.to_str().unwrap().to_owned());
+            let file_name = path.file_name().map(|file| file.to_string_lossy());
```

Comment:
I don't think it makes sense to include lossy results under completion since this won't result in a valid filepath anyway (and when you try to open the file you'll instead be making a new one). We should filter out these results instead